app/konnect/dev-portal/customization/index.md

```diff
@@ -29,6 +45,8 @@ To add a custom URL to Dev Portal, you need:
 * A domain and access to configure the domain's DNS `CNAME` records.
 * Your organization's auto-generated default Dev Portal URL.
 
+You can also choose to [self-host the Dev Portal with Netlify](/konnect/dev-portal/customization/netlify/).
```

Comment:
You can also choose to [self-host the Dev Portal with Netlify](/konnect/dev-portal/customization/netlify/) or any other static hosting service that supports single page applications.